app/js/components/orderEntry/OrdersTable.jsx

```diff
@@ -21,10 +29,114 @@ export class OrdersTable extends PureComponent {
     }
   }
 
+  getUUID = (items, itemName) => items.find(item => item.display === itemName)
+
+  setDiscontinuedDrugOrder = async (order) => {
+    const {
+      drugDispensingUnits,
+      drugDosingUnits,
+      drugRoutes,
+      durationUnits,
+      orderFrequencies,
+    } = this.props.allConfigurations;
+
+    const {
+      careSetting,
+      dose,
+      dosingUnit,
+      drugInstructions,
+      dosingType,
+      drug,
+      duration,
+      durationUnit,
+      frequency,
+      orderer,
+      dispensingQuantity,
+      dispensingUnit,
+      route,
+      type,
+      previousOrder,
+      orderNumber,
+    } = order;
+
+    const discontinueDecision = await swal("Are you sure you want to discontinue this order ?", {
+      buttons: {
+        YES: "YES",
+        NO: 'NO',
+      },
+    });
+    if (discontinueDecision === "YES") {
+      const reason = await swal({
+        text: 'Reason for discontinuing',
+        content: "input",
+        button: {
+          text: "DISCONTINUE!",
+          closeModal: true,
+        },
+      });
```

Comment:
@Efosaok on this PR ?

Reply:
yes, it's possible, am not saying it should be moved to `@openmrs-reactcomponents`, just have it in a separate file so it can be imported and used anywhere.

Reply:
@Efosaok so from the way the toastr is being used, it defines an error and success toast and both is being exported for their different use cases, the swal prompt here is not that specific in it's use case cos you will have to define the configuration for receiving and handling prompts responses in ways unimaginable, sorta like `window.alert`, so when you say it is possible, I'd be happy to hear your thoughts. What are you thinking ?